packages/fx-core/tests/component/generator/generator.test.ts

```diff
@@ -540,10 +573,33 @@ describe("Generate sample using download directory", () => {
     const axiosStub = sandbox.stub(axios, "get");
     axiosStub.onFirstCall().resolves({ status: 502 });
     axiosStub.onSecondCall().resolves({ status: 502 });
-    const result = await Generator.generateSample(ctx, tmpDir, "bot-sso");
+    const result = await Generator.generateSample(ctx, tmpDir, "test");
+    assert.isTrue(result.isErr());
+    if (result.isErr()) {
+      assert.equal(result.error.innerError.name, "DownloadSampleNetworkError");
+    }
+  });
+
+  it("throw error if one file download failed", async () => {
+    const axiosStub = sandbox.stub(axios, "get");
+    const sampleName = "test";
+    const mockFileName = "test.txt";
+    const mockFileData = "test data";
+    const fileInfo = [
+      { type: "file", path: `${sampleName}/${mockFileName}` },
+      { type: "file", path: `${sampleName}/${mockFileName}_1` },
+      { type: "file", path: `${sampleName}/${mockFileName}_2` },
+    ];
+    axiosStub.onFirstCall().resolves({ status: 200, data: { tree: fileInfo } });
```

Comment:
Is `onFirstCall` the same with `onCall(0)`? If so, I think using `onCall(0)` will reduce some confusion.

Reply:
Yes, let me update.